src/Form/Type/GiftCardConfigurationImageType.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusGiftCardPlugin\Form\Type;
+
+use Sylius\Bundle\CoreBundle\Form\Type\ImageType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class GiftCardConfigurationImageType extends ImageType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        parent::buildForm($builder, $options);
+
+        if ($options['remove_type']) {
+            $builder->remove('type');
```

Comment:
What is it that this does? :)

Reply:
It is to allow having the `backgroundImage` field in `GiftCardConfigurationType`  instead of collection of images where you have to enter the type